Makefile

```diff
@@ -49,3 +51,26 @@ tester-image: ## Build docker image for the tester component
 kind-load: image tester-image ## Load the required image to KinD cluster
 	kind load docker-image $(REPO):latest
 	kind load docker-image $(TESTER_REPO):v1
+
+.PHONY: images
+images: ## Build multi arch docker images for the project
+	docker buildx build \
+		--platform $(SUPPORTED_ARCHS) \
+		--output "type=image,push=false" \
+		--tag $(REPO):$(VERSION) \
+		--tag $(REPO):latest \
```

Comment:
I agree, instructions in the readme or directly in `make help` about how to build multi-arch images will be helpful.

Reply:
Maybe we can script it as a different make target (say, `make builder`), and mention in the README that if one encounters the above error message, run `make builder`. What do you think of something like:

```
.PHONY: builder
builder:
   docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
   docker buildx create --name=multiarch-builder --driver=docker-container --platform="${SUPPORTED_ARCHS}" --use
multiarch
   docker buildx inspect multiarch-builder --bootstrap
```

Reply:
That is also possible, I will try it out.

Reply:
@ihcsim 
is it necessary to run `docker run --rm --privileged multiarch/qemu-user-static --reset -p yes`?
because in this case, we don't use QEMU, we just utilize golang internal cross-compile features.

Reply:
AIUI, if the Dockerfile has the `RUN` commands, we will need the qemu in order for those commands to be executed during the multi-arch build.

Reply:
AFAICT, the buildx github action has to do the same thing. See https://github.com/crazy-max/ghaction-docker-buildx/blob/44fd46c7833e095f13bb1d8ace66bd821198bfef/src/main.ts#L25.